src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -58,17 +58,17 @@ public PersonCard(Person person, int displayedIndex) {
                 .sorted(Comparator.comparing(skill -> skill.skillName))
                 .forEach(skill -> skills.getChildren().add(new Label(skill.skillName)));
 
-        String jobIdentifier = person.getMatch();
-        if (jobIdentifier != null) {
+        String jobIdentifier = person.getMatch().orElse("Unemployed");
```

Comment:
Not sure if this counts as "null" checking

Reply:
I think you could just check `isEmpty()`